UPGRADE.md

```diff
@@ -7,6 +7,13 @@
 Write permissions for the webserver must be set on `web/uploads` instead of
 `web/uploads/media` alone to support simple cache clearing.
 
+### BlameSubscriber
+
+The BlameBehavior has been moved from the DocumentManager component to the
+Sulu Content component. Documents which implemented
+`Sulu\Component\DocumentManager\Behavior\Audit\BlameBehavior` should now
```

Comment:
Would be possible, but then we introduce a coupling from sulu/sulu to the document manager.